packages/ui/src/components/MenuV2/__stories__/Sentiments.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import type { StoryFn } from '@storybook/react'
+import { MenuV2 } from '..'
+import { DefaultDisclosure } from './Template.stories'
+
+export const Sentiments: StoryFn<typeof MenuV2> = () => (
+  <MenuV2 disclosure={DefaultDisclosure}>
+    <MenuV2.Item>default disabled</MenuV2.Item>
+    <MenuV2.Item sentiment="danger">Danger disabled</MenuV2.Item>
```

Comment:
You've switch disable texts for 'default' and 'danger' is this examples

Reply:
It's only borderless when there is an `href`, similar to `Menu` so I guess it's fine.